Packs/Core/Playbooks/playbook-Uncommon_creation_or_access_operation_of_sensitive_shadow_copy_by_a_high_risk_process_README.md

```diff
@@ -0,0 +1,64 @@
+This playbook addresses the following alerts:
+ 
+- Uncommon creation or access operation of sensitive shadow copy by a high-risk process
+ 
+Playbook Stages:
+  
+Triage: 
+ 
+- Check if the causality process image (CGO) is signed or not
+ 
+Investigation:
+ 
+- Examine process details, prevalence, and historical data for similar alerts
```

Comment:
historical data for similar alerts? what does it mean? what do you look for? need to be in a separated line